routes/v1-launches.js

```diff
@@ -10,7 +11,11 @@ v1.get("/latest", (req, res, next) => {
       if (err) {
         return next(err)
       }
-      res.json(doc)
+      if (doc.length == 0) {
+        res.status(404)
+        return res.json(error)
+      }
+      res.json(doc[0])
```

Comment:
This is significant, but probably makes more sense to return single item instead of array with single item.